que8.py

```diff
@@ -0,0 +1,13 @@
+# Define a class with a generator which can iterate the numbers, which are divisible by 7, between a given range 0 and n.
```

Comment:
**_# Define a class with a generator that can iterate the numbers, which are divisible by 7, between a given range 0 and n._**

The problem statement says that you have to define a class for the same.